### PR TITLE
corrected some minor errors and added a walkthrough for those running locally

### PR DESCRIPTION
**Pull request by IBRice101**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def challenge5():
     if (os.getenv("PORT") == '8080' and re.search('(curl|wget)\/*', request.headers.get('User-Agent').lower())):
         return "Challenge String: Vjgfqgug it a qugwvy dopo vrql\n"
     else:
-        return pick_one(errormsg) 
+        return pick_one(errormsg)
```
Comment (yeroc-sebrof) [inline app.py:88]: Did you run Python flake8, some other linter, or did your IDE do this?

File: README.md [hunk 2]
```diff
@@ -13,6 +25,8 @@ To get the encoding challenge for this one send a GET request to 192.168.22.100/
 
 `curl http://192.168.22.100/challenge1?whoAreYou=AHacker&areYouSure=yes`
 
+- **NOTE**: slashes are required in front of the `?` and `&` symbols (`\?` and `\&` respectively)
```
Comment (yeroc-sebrof) [inline README.md:28]: I'm pretty sure I copied exactly what worked in my terminal for this answer. What terminal did you use for this?

File: README.md [hunk 3]
```diff
@@ -2,8 +2,20 @@
 
 This lab should be easy enough, apply some CLI knowledge and the man page for curl to get the encoding/encryption challenges. Once you have the challenge you need to decode it to get the flag. Your target is the web server at 192.168.22.100
 
+Don't forget to use Curl's man page if you need some help!
+
 ---
 
+### Running locally
+
+If you would like to do this CTF locally, write the following commands on the command line:
```
Comment (yeroc-sebrof) [inline README.md]: I'd add a line to ensure that docker and docker-compose are installed. I'll do this later if you're busy

File: app.py (hunk 1 above)
Comment (IBRice101) [reply app.py:88]: My code linter in nvim flagged them up and I just thought I'd clean them, was going to make some further changes that the language server was suggesting but couldn't find a way to resolve warnings without errors appearing

File: README.md (hunk 2 above)
Comment (IBRice101) [reply README.md:28]: I'm using Alacritty on Manjaro Linux, if this is not standard in other terminals I'd be happy to remove or note the issue

File: README.md (hunk 3 above)
Comment (IBRice101) [reply README.md]: Done just now, see following commit :)

**Pull request merged**